iceoryx_examples/waitset/README.md

```diff
@@ -119,22 +119,23 @@ are stored inside of the **EventInfo** and can be acquired by the user.
 
 ## Use Cases
 This example consists of 5 use cases.
- 
- 1. `ice_waitset_gateway.cpp`: We build a gateway to forward data
-    to another network. A list of subscriber events are handled in an uniform way
-    by defining a callback which is executed for every subscriber who
-    has received data.
+ 1. `ice_waitset_basic`: A single subscriber is notified by the waitset if data arrives.
 
  2. `ice_waitset_grouping`: We would like to group multiple subscribers into 2 distinct
     groups and handle them whenever they have a specified state according to their group membership.
 
- 3. `ice_waitset_individual`: A list of subscribers where every subscriber is
+ 3. `ice_waitset_gateway.cpp`: We build a gateway to forward data
```

Comment:
Since you didn't change the order in the example description below, please keep `ice_waitset_gateway.cpp` as the second example.

Reply:
Something went wrong here...